_includes/example_table.html

```diff
@@ -1,3 +1,7 @@
+{% comment %}
+    How does this play with external_examples.html?
```

Comment:
I am not sure. I haven't had need to use external_examples internally. I am not sure if the stlab external examples still work, either.